### ISSUE TITLE
Cannot create Distribution Link (POW) for named user restricted object.

### ISSUE THREAD

**Issue by dkelsey** [bcgov/ckanext-bcgov#686]
# ENV
`PROD`, `CAT`, 
`CAD` un-verified.

At the time of writing this, both TEST and PROD are pointing to the PROD OFI API and PROD distribution.

# TESTCASE
* Create a Geographic Dataset
* fill in required fields
* add a named user **object_name**
I use `WHSE_FOREST_TENURE.BCTS_BLOCK_ACCESS_ROADS_SP` which I have confirmed is a named user dataset.
* click 'Save Dataset' to persist the state of the records --- this is kludgey but it is the process for now.
* re-open the dataset for editing
* click the **Resources** tab
* click the **'Manage BCGW Resources'** button to add distribution links

# EXPECTED
* a pop-up stated the resource exists and prompting do you wish to add resources.

# ACTUAL
* a pop-up stating 'Object is not available, please contact your administrator' 
![screen shot 2019-01-14 at 12 09 27 pm](https://user-images.githubusercontent.com/3906047/51138249-76dafc00-17f5-11e9-8f8e-2a4f52950661.png)

**Comment by mdunhamwilkie** [comment]
@dkelsey and @hleckenb.  It is actually possible to add the ofi object if you follow the originally-intended workflow of pressing Next: Add Data instead of Save Dataset.  This functionality was re-made operational in another ticket (https://github.com/bcgov/ckanext-bcgov/issues/641)

**Comment by mdunhamwilkie** [comment]
[ofi_add_works_for_me.docx](https://github.com/bcgov/ckanext-bcgov/files/2760064/ofi_add_works_for_me.docx)

**Comment by dkelsey** [comment]
Geographic dataset creation, and the process of configuring distribution resources has always been problematic and cumbersome.  We require that Editors create a metadata record, before the data is even available in PROD and before distribution has been configured.  Editors were asked to add 'dummy' resources, the FAQ, then after the data is loaded and distibution is correctly configured, we instruct editors to re-visit the record and configure the distribution links.  

in short, the "happy path" is not a single end-to-end record creation and distribution link creation session.  It has never happened.  

the steps that are failing now, used to work.  

Further, when I now press the 'Next: Add Data' button, I'm taken out of edit mode and returned to the unpublished record.

**Comment by dkelsey** [comment]
I followed the instructions in your ofi workflow.  I created a new record and followed the step.
it did not work for me.  I got the same pop-up.
